lua/cfc_ulx_commands/timed_punishments/server/data.lua

```diff
@@ -0,0 +1,119 @@
+local function escapeArg( arg )
+    if arg == SQL_NULL then
+        return "NULL"
+    elseif type( arg ) == "number" then
+        return arg
+    else
+        return sql.SQLStr( arg )
+    end
+end
+
+local function queryFormat( query, ... )
+    local args = {}
+    for i, arg in ipairs{ ... } do
```

Comment:
Do we have a guideline for `func( { ... } )` and `func{ ... }`? If not we should add that... For the record, I like the way you did it; but it could be confusing for new developers...?

Reply:
I copied this over from some code that Sam or PCL made, but I think you're right - I'll keep it consistent with the style guidelines ([ref](https://github.com/CFC-Servers/cfc_glua_style_guidelines#spaces-inside-parentheses-and-curly-braces-if-they-contain-content))

For the record though, later in the ULX code it uses this pattern, but I give it a pass because that's the standard style for ULX commands